gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/JobCatalog.java

```diff
@@ -77,22 +82,38 @@ default StandardMetrics getStandardMetrics() {
     @Getter private final ContextAwareCounter numAddedJobs;
     @Getter private final ContextAwareCounter numDeletedJobs;
     @Getter private final ContextAwareCounter numUpdatedJobs;
-
-    public StandardMetrics(final JobCatalog parent) {
-      this.numAddedJobs = parent.getMetricContext().contextAwareCounter(NUM_ADDED_JOBS);
-      this.numDeletedJobs = parent.getMetricContext().contextAwareCounter(NUM_DELETED_JOBS);
-      this.numUpdatedJobs = parent.getMetricContext().contextAwareCounter(NUM_UPDATED_JOBS);
-      this.numActiveJobs = parent.getMetricContext().newContextAwareGauge(NUM_ACTIVE_JOBS_NAME,
+    @Getter private final ContextAwareTimer timeForJobCatalogGet;
+    @Getter private final ContextAwareHistogram histogramForJobAdd;
+    @Getter private final ContextAwareHistogram histogramForJobUpdate;
+    @Getter private final ContextAwareHistogram histogramForJobDelete;
+
+    public StandardMetrics(final JobCatalog jobCatalog) {
+      this.timeForJobCatalogGet = jobCatalog.getMetricContext().contextAwareTimerWithSlidingTimeWindow(TIME_FOR_JOB_CATALOG_GET, 1, TimeUnit.MINUTES);
```

Comment:
We could assign `jobCatalog.getMetricContext()` to a local variable, making the code a bit cleaner.